src/components/form/ReviewStep.js

```diff
@@ -14,53 +14,64 @@ const ReviewStep = () => {
   const { getState } = useFormApi();
   return (
     <TextContent>
-      <Text component={TextVariants.small}>
+      <Text>
         Review the information and click the Create button to create your image
         using the following criteria.
       </Text>
-      <Text component={TextVariants.h3}>Image output</Text>
+      <Text component={TextVariants.h1}>Image output</Text>
       <TextList
         component={TextListVariants.dl}
         data-testid="review-image-output"
       >
         <TextListItem component={TextListItemVariants.dt}>Release</TextListItem>
         <TextListItem component={TextListItemVariants.dd}>
-          Add value
+          Red Hat Enterprise Linux (RHEL) 8.3
```

Comment:
Same as above, this should be the value selected in first step

Reply:
DONE